backend/src/controllers/questionGroup.js

```diff
@@ -36,4 +36,32 @@ const getQuestionGroups = async (req, res) => {
   res.status(200).send({ data: questionGroups });
 };
 
-module.exports = { getQuestionGroups };
+const getQuestionGroupsByLessonId = async (req, res) => {
+  const { lessonId } = req.params;
+  const lesson = await db.Lesson.findByPk(lessonId);
```

Comment:
eigenlijk moet hier nog wel n check bij dat de req.teacher daadwerkelijk een teacher is van de lessomn